### PR TITLE
improvement(sct-grafana-dashboard): added new 3.3 json

### PR DESCRIPTION
**Pull request by bentsi**
Since monitoring version was not changed  based it on 3.2.
(master dashboard is stale)

### CODE REVIEW

**Comment by fruch** [comment]
@bentsi I think we should also have a task to fix the code to just send it as critical event, and us the master dashborad

**Comment by bentsi** [comment]
@fruch I think Alex fixed this in #1774 , Right Alex?

File: data_dir/scylla-dash-per-server-nemesis.3.3.json [hunk 1]
```diff
@@ -0,0 +1,5413 @@
...
+                        "id": 30,
+                        "limit": 10,
+                        "links": [],
+                        "query": "",
+                        "recent": false,
+                        "search": true,
+                        "span": 2,
+                        "starred": false,
+                        "tags": [
+                            "master"
```
Comment (yarongilor) [inline data_dir/scylla-dash-per-server-nemesis.3.3.json]: might better be changed to "3.3"

**Comment by aleksbykov** [comment]
@bentsi , yes in #1774 should fix common case for future version

File: data_dir/scylla-dash-per-server-nemesis.3.3.json (hunk 1 above)
Comment (bentsi) [reply data_dir/scylla-dash-per-server-nemesis.3.3.json]: where is this tag used?

Comment (bentsi) [reply data_dir/scylla-dash-per-server-nemesis.3.3.json]: This is a group of dashboards for master. It should be there
image

**Pull request merged**